stock_transport_multi_address/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
You should use git mv instead manually move file.

git mv stock_transport_multi_address/model/res_partner.py stock_transport_multi_address/models/